Python/Product/PyDebugAttach/PyDebugAttach.cpp

```diff
@@ -1178,14 +1199,15 @@ bool DoAttach(HMODULE module, ConnectionInfo& connInfo, bool isDebug) {
                         // all of the work here needs to be minimal - in particular we shouldn't
                         // ever evaluate user defined code as we could end up switching to this
                         // thread on the main thread and corrupting state.
-                        auto prevThreadState = getThreadTls(threadStateIndex);
```

Comment:
There was a warning that the result is not used, and I'm assuming that this function has no side-effect and it's pointless to call it if we don't use the result.